progress.go

```diff
@@ -86,34 +85,41 @@ func (p *Progress) AddBar(total int) *Bar {
 // Listen listens for updates and renders the progress bars
 func (p *Progress) Listen() {
 	p.lw.Out = p.Out
-	for {
-		select {
-		case <-p.stopChan:
-			return
-		default:
-			time.Sleep(p.RefreshInterval)
+
+	go func() {
```

Comment:
exported function `.Listen()` changes its behavior from doing the listen loop to spawning a goroutine and exiting without waiting for the end of the processing. Which is a breaking change, IMHO and should probably avoided.